modules/ocf_backups/files/create-encrypted-backup

```diff
@@ -23,30 +39,40 @@ fi
 cleanup() {
     if [ -e "$SNAPSHOT_PATH" ]; then
         echo "Removing logical volume: $SNAPSHOT_PATH"
-        lvremove -f "$SNAPSHOT_PATH"
+        /sbin/lvremove -f "$SNAPSHOT_PATH"
     fi
 }
 trap cleanup EXIT
 
 # we don't have *quite* enough space to make a snapshot with the full 2TB of
 # disk space, so we make a large enough snapshot that we'll *probably* avoid
 # trouble
-lvcreate -L 500G --snapshot --name "$SNAPSHOT_PATH" "$TARGET_VOLUME"
+/sbin/lvcreate -L 500G --snapshot --name "$SNAPSHOT_PATH" "$TARGET_VOLUME"
 
 umask 077
 rm -rf "$ARCHIVE_DIRECTORY"
 mkdir "$ARCHIVE_DIRECTORY"
 
-dd if="$SNAPSHOT_PATH" bs=32M | pigz |
-    gpg --homedir "$gpgdir" \
-        --quiet \
-        --encrypt \
-        --trust-model always \
-        --compress-algo none \
-        --recipient ckuehl@berkeley.edu \
-        --recipient willh@ocf.berkeley.edu \
-        --recipient jvperrin@ocf.berkeley.edu \
-        --recipient mattmcal@ocf.berkeley.edu |
-    pv |
+compress_and_encrypt() {
+    dd if="$SNAPSHOT_PATH" bs=32M | pigz |
+        gpg --homedir "$gpgdir" \
+            --quiet \
+            --encrypt \
+            --trust-model always \
+            --compress-algo none \
+            --recipient ckuehl@berkeley.edu \
+            --recipient willh@ocf.berkeley.edu \
+            --recipient jvperrin@ocf.berkeley.edu \
+            --recipient mattmcal@ocf.berkeley.edu
+}
+
+split_parts() {
     split -d -a 4 -b 5G - \
         "${ARCHIVE_DIRECTORY}/backup-$(date +%Y-%m-%d).img.gz.gpg.part"
+}
+
+if [ "$QUIET" = true ]; then
+    compress_and_encrypt | split_parts
+else
+    compress_and_encrypt | pv | split_parts
+fi
```

Comment:
I was originally trying to just add the `-q` flag to `pv` if the variable was set, but that didn't work so well, so I went with this instead. I should really use bash functions more in scripts, they are pretty nice.